RestCommClient/Classes/Push/PushHandler.m

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
Have you considered introducing testing facilities for the PushHandler (provisioning) functionality? Seems like a good candidate

Reply:
Nope. If we need to introduce testing, I will need to organize the code differently. I am for it if we have time.